spec/models/good_spec.rb

```diff
@@ -4,4 +4,40 @@
   describe 'associations' do
     it { should belong_to(:basket) }
   end
+
+  describe 'tax calculation' do
+    before :each do
+      @importexampt = Good.new(name: 'imported chocolat', quantity: '1', price: '10.00')
+      @imported = Good.new(name: 'imported', quantity: '1', price: '47.50')
+      @exampt = Good.new(name: 'book', quantity: '1', price: '12.49')
+      @basic = Good.new(name: 'music', quantity: '1', price: '14.99')
+    end
+
+    after :each do
+      @importexampt.destroy
+      @imported.destroy
+      @exampt.destroy
+      @basic.destroy
+    end
```

Comment:
Same here: factories will take care to create themselves and clean up after the spec, making your tests more readable, and you are free from doing this house cleaning.